website/docs/r/bot_channel_direct_line_speech.html.markdown

```diff
@@ -55,6 +55,8 @@ The following arguments are supported:
 
 * `bot_name` - (Required) The name of the Bot Resource this channel will be associated with. Changing this forces a new resource to be created.
 
+* `cognitive_account_id` - (Optional) The resource ID of the Cognitive Account.
```

Comment:
we don't use the phrase `Resource ID` since it's ambigous between Azure and Terraform - can we make this:

```suggestion
* `cognitive_account_id` - (Optional) The ID of the Cognitive Account this Bot Channel should (be associated with|use|etc?).
```

Reply:
Updated